test/integration/puppetlabs/services/certificate_authority/certificate_authority_int_test.clj

```diff
@@ -278,225 +278,209 @@
+            ca-cert (str bootstrap/master-conf-dir "/ca/ca_crt.pem")
+            ca-key (str bootstrap/master-conf-dir "/ca/ca_key.pem")
+            ca-crl (str bootstrap/master-conf-dir "/ca/ca_crl.pem")]
+          (bootstrap/with-puppetserver-running-with-mock-jrubies
+           "JRuby mocking is safe here because all of the requests are to the CA
+           endpoints, which are implemented in Clojure."
+            app
+            {:certificate-authority {:enable-infra-crl true}}
+            (testing "should update infrastructure CRL"
+              (let [ca-cert' (ssl-utils/pem->ca-cert ca-cert ca-key)
+                    cm-cert (utils/pem->cert (ca/path-to-cert signed-dir subject))
+                    node-cert (utils/pem->cert (ca/path-to-cert signed-dir node-subject))
+                    options {:ssl-cert ca-cert
+                             :ssl-key ca-key
+                             :ssl-ca-cert ca-cert
+                             :as :text}
+                    cert-status-request (fn [action
+                                             certtorevoke]
+                                          (http-client/put
+                                           (str "https://localhost:8140/"
+                                                "puppet-ca/v1/certificate_status/"
+                                                certtorevoke)
+                                           (merge options
+                                                  {:body (str "{\"desired_state\": \""
+                                                              action
+                                                              "\"}")
+                                                   :headers {"content-type"
+                                                             "application/json"}})))]
+                (testing "Infra CRL should contain the revoked compile master certificate"
+                  (let [revoke-response (cert-status-request "revoked" subject)]
+                    ;; If the revocation was successful infra CRL should contain above revoked compile master cert
+                    (is (= 204 (:status revoke-response)))
+                    (is (utils/revoked? (utils/pem->ca-crl infra-crl ca-cert') cm-cert))))
 
-              (testing "Infra CRL should NOT contain a revoked non compile master certificate"
-                (let [revoke-response (cert-status-request "revoked" node-subject)]
-                  (is (= 204 (:status revoke-response)))
-                  (is (not (utils/revoked? (utils/pem->ca-crl (str master-conf-dir "/ssl/ca/infra_crl.pem") ca-cert) node-cert)))))))
+                (testing "Infra CRL should NOT contain a revoked non compile master certificate"
+                  (let [revoke-response (cert-status-request "revoked" node-subject)]
+                    (is (= 204 (:status revoke-response)))
+                    (is (not (utils/revoked? (utils/pem->ca-crl infra-crl ca-cert') node-cert)))))))
```

Comment:
This is failing for me locally and I have no idea why

Reply:
I'll give it a shot.

Reply:
It fails for me too.